src/robot/libraries/Collections.py

```diff
@@ -1034,4 +1206,4 @@ def _get_matches_in_iterable(iterable, pattern, case_insensitive=False,
                       spaceless=is_truthy(whitespace_insensitive),
                       regexp=regexp)
     return [string for string in iterable
-            if is_string(string) and matcher.match(string)]
+            if is_string(string) and matcher.match(string)]
```

Comment:
Newline missing at the end of the file. This has actually caused a conflict because the above row was changed. That underlines one reason to always end files with newlines. Luckily fixing the conflict is easy.